packages/app/src/specs/SpecHeaderCloudDataTooltip.vue

```diff
@@ -207,36 +172,16 @@ const projectConnectionStatus = computed(() => {
   if (props.gql.currentProject?.cloudProject?.__typename === 'CloudProjectNotFound') return 'NOT_FOUND'
 
   if (props.gql.currentProject?.cloudProject?.__typename === 'CloudProjectUnauthorized') {
-    if (hasRequestedAccess.value) {
-      return 'ACCESS_REQUESTED'
-    }
-
     return 'UNAUTHORIZED'
   }
 
   return 'CONNECTED'
 })
 
-const requestAccessMutation = useMutation(SpecHeaderCloudDataTooltip_RequestAccessDocument)
-
-async function requestAccess () {
-  const projectId = props.gql.currentProject?.projectId
-
-  if (projectId) {
-    const result = await requestAccessMutation.executeMutation({ projectId })
-
-    if (result.data?.cloudProjectRequestAccess?.__typename === 'CloudProjectUnauthorized') {
-      hasRequestedAccess.value = result.data.cloudProjectRequestAccess.hasRequestedAccess ?? false
-    } else {
-      hasRequestedAccess.value = false
-    }
-  }
-}
-
 const tooltipTextKey = computed(() => {
   if (projectConnectionStatus.value === 'CONNECTED') return VALUES[props.mode].connected
 
-  if (['UNAUTHORIZED', 'ACCESS_REQUESTED'].includes(projectConnectionStatus.value)) return VALUES[props.mode].noAccess
+  if (['UNAUTHORIZED'].includes(projectConnectionStatus.value)) return VALUES[props.mode].noAccess
```

Comment:
Probably not worth using an array and includes here since it's only one value now